include/mbedtls/config.h

```diff
@@ -1706,6 +1706,22 @@
  */
 #define MBEDTLS_SSL_PROTO_TLS1_2
 
+/**
+ * \def MBEDTLS_SSL_PROTO_TLS1_3_EXPERIMENTAL
+ *
+ * This is an experimental macro used to selectively enable parts
+ * of the code that solely contribute to the ongoing development of
+ * a prototype implementation for TLS 1.3 and DTLS 1.3 but aren't
+ * used otherwise.
+ *
+ * \warning Features under the control of this macro are experimental
+ *          and don't come with any stability guarantees.
+ *
+ * Uncomment this macro to enable experimental and partial
+ * functionality specific to TLS 1.3.
+ */
+//#define MBEDTLS_SSL_PROTO_TLS1_3_EXPERIMENTAL
```

Comment:
Observation: I like `MBEDTLS_SSL_PROTO_TLS1_3_EXPERIMENTAL` having the word `EXPERIMENTAL` in it, as it will deter people from trying it out too soon. However, when we rename it to `MBEDTLS_SSL_PROTO_TLS1_3`, there will be a lot of searching and replacing and fiddling with broken formatting, which will be a really tedious task.

Reply:
Hm, I don't follow - it's going to be a straightforward `sed`, or not?

Reply:
I agree, replacement is easily done with `sed`. I also agree with Simon that dumb replacement is in general not enough as it can leave the formatting broken (line length, vertical alignment). However in this specific case I think this unlikely to be an issue as most uses are going to be
```
#if defined(MBEDTLS_SSL_PROTO_TLS1_3_EXPERIMENTAL)
```
on a line on its own. The few instances where it's not the case can by fixed manually if necessary (which is made less likely by the fact that we'd be changing to a shorter name).

So overall, I think the benefits of having `_EXPERIMENTAL` in the name outweigh the potential costs.

Reply:
No action here, then.